### PR TITLE
(Do not merge) Edit repository names 

### PR DESCRIPTION
**Pull request by clausmullie**
Space to work together on repository subtitles. Not to be merged. This pull request and the file it created should be deleted once the text has been finalised and implemented.

### CODE REVIEW

File: profile/Repository names [hunk 1]
```diff
@@ -0,0 +1,19 @@
+blog: Home to the Blog for Public Code and where blog posts are planned and collectively written.
```
Comment (clausmullie) [inline profile/Repository names]:
```suggestion
blog: Home to the Blog for Public Code, where blog posts are planned and collectively written.
```

File: profile/Repository names [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+governance-game: Card game used to learn what needs of governance organizations collaborating on public code codebases has.
+
+projects: Home to a list of useful resources developed by the Foundation for Public Code.
+
+codebasestewards: An issue-only repository, containing the issues codebase stewards are actively working on, that does not ft in other repositories. These are used on the weekly planning board.
+
+bumperscripter: Script to generate video podcast bumper animations using the Processing framework. Makes it easy to create animations where only a few text strings change every episode (like guest names etc.).
+
+playground: A repository for our staff to learn, play and experiment with git. Not meant for reuse.
```
Comment (clausmullie) [inline profile/Repository names]:
```suggestion
playground: A repository for our staff to learn, play and experiment with git. Not intended for reuse.
```

**Comment by ericherman** [comment]
I see it is draft; it all looks reasonable to me, so far.

File: profile/Repository names [hunk 3]
```diff
@@ -0,0 +1,19 @@
+blog: Home to the Blog for Public Code and where blog posts are planned and collectively written.
+
+standard: The Standard for Public Code - a model for public organizations to build their own open source solutions to enable successful future reuse by similar public organizations in other places.
+
+jitsi-community: A tool that allows users to sign up to a Jitsi instance that is not open to everyone, using a web UI so that they can start sessions by themselves.
```
Comment (clausmullie) [inline profile/Repository names]:
```suggestion
jitsi-community: A tool that allows community members to host and run video calls on our self-hosted Jitsi instance. This tool enables users to sign up, create rooms, and start calls as a moderator.
```

File: profile/Repository names [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+illustrations: A collection of illustrations meant to be reused in the communications of the Foundation for Public Code.
```
Comment (clausmullie) [inline profile/Repository names]:
```suggestion
illustrations: A collection of illustrations to support the communications of the Foundation for Public Code.
```

File: profile/Repository names [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+publiccode.net: The website of the Foundation for Public Code that is the most public facing and the primary place for people to learn about the organization.
```
Comment (clausmullie) [inline profile/Repository names]:
```suggestion
publiccode.net: Home to the website of the Foundation for Public Code. This is the most public-facing place for people to learn about the organization.
```

File: profile/Repository names [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+governance-game: Card game used to learn what needs of governance organizations collaborating on public code codebases has.
```
Comment (clausmullie) [inline profile/Repository names]:
```suggestion
governance-game: A card game that helps organizations and communities explore governance around a shared codebase, whether hypothetical or in a real-world collaboration.
```

File: profile/Repository names [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+projects: Home to a list of useful resources developed by the Foundation for Public Code.
```
Comment (clausmullie) [inline profile/Repository names:13]:
```suggestion
projects: Home to a list of useful projects developed by the Foundation for Public Code.
```
Perhaps I'm too About'ized, but I think of About as a place with resources. What is unique about projects? Time bound? In collaboration with other organizations? Anything we could add here as a qualifier?

File: profile/Repository names [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+codebasestewards: An issue-only repository, containing the issues codebase stewards are actively working on, that does not ft in other repositories. These are used on the weekly planning board.
```
Comment (clausmullie) [inline profile/Repository names]:
```suggestion
codebasestewards: An issue-only repository with issues the codebase stewards are actively working on, but that do not ft in other repositories. These are used on the weekly planning board.
```

File: profile/Repository names [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+
+bumperscripter: Script to generate video podcast bumper animations using the Processing framework. Makes it easy to create animations where only a few text strings change every episode (like guest names etc.).
```
Comment (clausmullie) [inline profile/Repository names]:
```suggestion
bumperscripter: Script to easily create video bumper animations for the podcast. It enables users to easily edit changing text strings (such as guest names) in an existing template. It uses the Processing framework.
```
Is template the right word here? Not sure.

File: profile/Repository names (hunk 3 above)
Comment (Ainali) [reply profile/Repository names]: No, this tool could be used on any self-hosted Jitsi instance, not only ours. But if the "our" is changed to "a" it is good.

File: profile/Repository names (hunk 7 above)
Comment (Ainali) [reply profile/Repository names:13]: Relevant discussion is in https://github.com/publiccodenet/projects/issues/10 which I think makes a good case for referring to them as resources.

File: profile/Repository names (hunk 9 above)
Comment (Ainali) [reply profile/Repository names]: This is perhaps better:
Script to easily create bumper animations for live video streams. It enables users to easily edit changing text strings (such as guest names) through a simple templating setup. It uses the Processing framework.

File: profile/Repository names (hunk 3 above)
Comment (clausmullie) [reply profile/Repository names]:
```suggestion
jitsi-community: A tool that allows users to sign up, create and run video calls on a self-hosted Jitsi instance. We use this to allow our community members to run calls on our infrastructure.
```

Aha! Got it. Does this fit better?

File: profile/Repository names (hunk 7 above)
Comment (clausmullie) [reply profile/Repository names:13]: Yes indeed!

File: profile/Repository names (hunk 9 above)
Comment (clausmullie) [reply profile/Repository names]: Indeed

Comment (clausmullie) [reply profile/Repository names]:
```suggestion
bumperscripter: Script to easily create bumper animations for live video streams. It enables users to easily edit changing text strings (such as guest names) through a simple templating setup. It uses the Processing framework.
```

**Comment by clausmullie** [comment]
These have now been implemented :tada:

**Pull request closed without merging**